0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -318,3 +187,4 @@ Content-Type: application/json
 }
 ```
 [Try this request](https://red.ilpdemo.org/.well-known/webfinger?resource=acct%3Abob%40red.ilpdemo.org)
```

Comment:
bob@red.ilpdemo.org doesn't exist, alice@red.ilpdemo.org does, though. You could either change this link or create the bob account so that it starts working :)